src/main/java/thaumicboots/api/IMeteor.java

```diff
@@ -0,0 +1,4 @@
+package thaumicboots.api;
+
+public interface IMeteor {
+}
```

Comment:
same

Reply:
Same as the previous response

Reply:
Look at the event handler and it'll make more sense

Reply:
no it doesn't. See this : `item instanceof IMeteor && !(item instanceof IComet))` this is litterally caused by your void walker.

Reply:
done